service/src/main/java/org/broadinstitute/listener/config/AppConfiguration.java

```diff
@@ -41,6 +41,7 @@ public TargetResolver targetResolver() {
   @Bean
   public SamResourceClient samResourceClient(TokenChecker tokenChecker) {
     return new SamResourceClient(
+        properties.getSetDateAccessedInspectorProperties().workspaceId(),
```

Comment:
I think this might be meant to be `getSamInspectorProperties()`!

Reply:
I had initially thought of the same property to use. But it seems that based on [README](https://github.com/DataBiosphere/terra-azure-relay-listeners#configuration-properties-1) and [application.yaml](https://github.com/DataBiosphere/terra-azure-relay-listeners/blob/4ddf0352e890a6c794cc8835056265500205ab46/service/src/main/resources/application.yml#L30-L34) `samInspectorProperties` doesn't have a workspace id in it. The only config that had the workspace id was [setDateAccessedInspectorProperties](https://github.com/DataBiosphere/terra-azure-relay-listeners/blob/4ddf0352e890a6c794cc8835056265500205ab46/service/src/main/resources/application.yml#L24-L28).

Reply:
Hm, ok. Tech debt :(